iphone/Maps/Core/Location/MWMLocationManager.mm

```diff
@@ -34,6 +34,48 @@ typedef NS_OPTIONS(NSUInteger, MWMLocationFrameworkUpdate) {
   BicycleRouting
 };
 
+static std::string StringFromGeoMode(GeoMode geoMode) {
```

Comment:
We name functions `DebugPrint` for such task, after that these variables can be logged naturally by macro, like:

`LOG(LINFO, ("geoMode:", geoMode));`